spec/services/manage_donation_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe ManageDonation do
+  # Because ManageDonation extends ManageAction, we won't re-test any of the logic that we didn't overwrite.
+  # So the only thing to test here is that the parameter dictionary is built correctly and sent to the queue
+  # correctly
+
+  let(:page) { create(:page) }
+  let(:data) { { email: 'bob@example.com', page_id: page.id } }
+
+  let(:full_donation_options) {
+    {
+        donationpage: {
+            name: 'donation',
+            payment_account: 'Default Import Stub'
+        },
+        order: {
+            amount: '1',
+            card_num: '4111111111111111',
```

Comment:
if `card_code` is CVV I'm not sure we can or should be passing that (It's [deprecated on BrainTree](https://developers.braintreepayments.com/reference/request/customer/create/ruby#credit_card.cvv)). If you have the ActionKit documentation page for this stuff somewhere in handy, could you link it to me? I'm not used to navigating their docs and having some difficulties figuring out what I should be looking at.